spec/features/page_feature_spec.rb

```diff
@@ -44,9 +42,9 @@ module Alchemy
         end
 
         context "if no language params are given" do
-          it "should redirect to url with nested language code" do
-            visit "/#{public_page_1.urlname}"
-            expect(page.current_path).to eq("/#{public_page_1.language_code}/#{public_page_1.urlname}")
+          it "doesn't prepend the url with the locale string" do
```

Comment:
Only if `@page.language.code` is the `I18n.default_locale`. If the page language is not the default locale, we need to redirect with prepended locale prefix.